tests/Doctrine/Tests/TestUtil.php

```diff
@@ -74,6 +78,34 @@ public static function getPrivilegedConnection(): DbalExtensions\Connection
         return $connection;
     }
 
+    public static function configureProxies(Configuration $configuration): void
+    {
+        $configuration->setProxyDir(__DIR__ . '/Proxies');
+        $configuration->setProxyNamespace('Doctrine\Tests\Proxies');
+
+        $proxyImplementation = $_ENV['ORM_PROXY_IMPLEMENTATION']
+            ?? $_SERVER['ORM_PROXY_IMPLEMENTATION']
```

Comment:
Some SAPIs store environment variables in `$_ENV`, some in `$_SERVER`. But since we're not dealing with any SAPI other than CLI, I can probably remove it.

Reply:
In fact, on CLI, environment variables are stored in `$_SERVER`. I think I might just switch to `getenv()`. 🤦🏻

Reply:
I remember stuggling with that myself in the past, and it seems to be quite a mess: https://www.php.net/manual/en/ini.core.php#ini.variables-order